Packages/tests/Basic/UTF_Utils.ipf

```diff
@@ -2485,6 +2485,88 @@ Function GetListOfObjectsWorks2()
 	CHECK_EQUAL_STR(result, expected)
 End
 
+Function GetListOfObjectsWorksTypeFlag()
+
+	string result, expected
+
+	NewDataFolder test
+	NewDataFolder :test:test2
+	NewDataFolder :test:test2:test3
+
+	DFREF dfr = $":test"
+	CHECK(DataFolderExistsDFR(dfr))
+
+	Make dfr:wv1
+	Make dfr:wv2
+	variable/G   dfr:var1
+	variable/G/C dfr:var2
+	string/G     dfr:str1
+
+	DFREF dfrDeep = $":test:test2"
+	CHECK(DataFolderExistsDFR(dfrDeep))
+
+	Make dfrDeep:wv3
+	Make dfrDeep:wv4
+	variable/G   dfrDeep:var3
+	variable/G/C dfrDeep:var4
+	string/G     dfrDeep:str2
+
+	result   = GetListOfObjects(dfr, ".*", recursive = 0, typeFlag = COUNTOBJECTS_DATAFOLDER)
```

Comment:
Wouldn't that be something for a MD test case?

Reply:
Yeep will do.

Reply:
Fixed.